narayana/ArjunaJTA/jta/pom.xml

```diff
@@ -199,6 +200,11 @@
             <artifactId>artemis-journal</artifactId>
             <version>${version.org.apache.activemq}</version>
         </dependency>
+		<dependency>
+		    <groupId>jakarta.transaction</groupId>
+		    <artifactId>jakarta.transaction-api</artifactId>
+		    <version>${version.jakarta.transaction-api}</version>
```

Comment:
We should be using the new bom otherwise the versions will start diverging again thus nullifying the benefits of using boms.

Reply:
I added the bom in performance